### PR TITLE
push my task files

### PR DESCRIPTION
**Pull request by Perpee**
Please review my work @promise-J and @ChidiPatrick and @Pieta9221

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
\ No newline at end of file
+## Men's Fashion Hub -  https://perpe123.github.io/Lesson_3_task_3_Obiefuna_Toochi/
+
+##How to run/test this project
+
+##ensure you have git installed properly on your machine
+
+##ensure you have node js installed in your machine
+
+##open your terminal and run "git clone "
```
Comment (Pieta9221) [inline README.md:9]: Add repo link

**Review by Pieta9221 — COMMENTED** [review]
Nice job :+1: :+1: :+1:
However this aspect of the task was not implemented:
Use CSS to create a custom typography for your web page. Experiment with different font families, sizes, styles, weights, and line-heights to create a visually appealing layout for your web page.
Kindly fix this before merging

**Comment by Perpee** [comment]
Ok thanks Pieta...tho I thought i did that

**Review by ChidiPatrick — APPROVED** [review]
Great job!

**Pull request merged**